README.md

```diff
@@ -295,19 +295,21 @@ HTTP/1.1 204 No Content
 Request:
 
 ```sh
-curl --include --request POST http://localhost:3000/sign-up \
+curl http://localhost:4741/sign-up \
+  --include \
+  --request POST \
   --header "Content-Type: application/json" \
   --data '{
     "credentials": {
-      "email": "an@example.email",
-      "password": "an example password",
-      "password_confirmation": "an example password"
+      "email": "'"${EMAIL}"'",
```

Comment:
I can see how these should be valid curls requests with response.

I was trying to make clear how the scripts are used.  I'm not sure how to do that if the examples don't contain the variables.  People frequently hard code the scripts and end up spending significant time debugging the wrong TOKEN/ID/etc.

Do you have a suggestion for accomplishing both goals?